### PR TITLE
Grass PROTO

### PR DESCRIPTION
**Pull request by fabienrohrer**
The Grass PROTO is now required for #772 and #761

- [x] Grass.proto with 4 types
- [x] Icon
- [x] Log

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: projects/appearances/protos/Grass.proto [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+{
+  PBRAppearance {
+    baseColor IS colorOverride
+    baseColorMap ImageTexture {
+      url [
+        %{= "\"textures/grass/grass_" .. fields.type.value .. "_base_color.jpg\"" }%
+      ]
+    }
+    roughness 1
+    %{ if typeID == "prickly" then }%
```
Comment (DavidMansolino) [inline projects/appearances/protos/Grass.proto]: `typeID` is never defined

Comment (fabienrohrer) [inline projects/appearances/protos/Grass.proto]:
```suggestion
    %{ if fields.type.value == "prickly" then }%
```

Comment (fabienrohrer) [reply projects/appearances/protos/Grass.proto]: Thank you, this is indeed wrong (silently ignored).

**Comment by fabienrohrer** [comment]
👍

**Comment by fabienrohrer** [comment]
New grass appearances:

| Name | Result |
| --- | --- |
| Maintained | grass_maintained |
| Artificial | grass_artificial |
| Mossy | grass_mossy |
| Prickly | grass_prickly |

**Pull request merged**